android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -827,6 +839,8 @@ internal class BetterPlayer(
         surface?.release()
         exoPlayer?.release()
     }
+    
+    val duration : Long? get() = exoPlayer?.duration
```

Comment:
Actually there is `getDuration()` at [here](https://github.com/dwango-nfc/betterplayer/blob/37ca8566fb718e5ea97cea60aab14a95df367f8f/android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt#L626) .
Do you know it?

I also will use it at this [PR](https://github.com/dwango-nfc/betterplayer/pull/49/files).

Reply:
@HiroshiOshiro Thanks, I didn't know that 😅